x-pack/legacy/plugins/graph/public/app.js

```diff
@@ -48,29 +48,18 @@ import { GraphVisualization } from './components/graph_visualization';
 
 import gws from './angular/graph_client_workspace.js';
 import { SavedWorkspacesProvider } from './angular/services/saved_workspaces';
-import {
```

Comment:
There's no more usage of 'fatalError' on line26
You can remove:
`import { KibanaParsedUrl } from 'ui/url/kibana_parsed_url';`

Reply:
Ah good point, but fatalError _should_ be used. I forgot to pull that into the persistence saga, will fix

Reply:
On second thought, throwing a fatal error if saving doesn't work is not a good user experience (and might even lead to unnecessary data loss). I changed the behavior to just show an error toast so the user can try again